docs/extending/README.md

```diff
@@ -70,7 +70,7 @@ snippet. In terms of the TypeScript typings from the NPM package, `activate` ret
   * Versions: `v1`
 * [Cluster Explorer API](clusterexplorer.md) - used for working with the Clusters tree
   * ID: `clusterexplorer`
-  * Versions: `v1`
+  * Versions: `v1`, `v1_1`
```

Comment:
Wondering if this should be `v1.1` instead of `v1_1`.  We have to use the underscore in the JavaScript names because the dot has special meaning, but the way people would normally refer to it is `1.1`.